plugins/org.eclipse.xtext/src/org/eclipse/xtext/serializer/impl/Serializer.java

```diff
@@ -151,6 +173,13 @@ protected EObject getContext(EObject semanticObject) {
 		return contexts.next();
 	}
 
+	protected IContext getIContext(EObject semanticObject) {
```

Comment:
I don't like the method name though I see that `getContext` was already taken. `getSerializationContext`?